autodistill/core/custom_detection_model.py

```diff
@@ -26,6 +31,28 @@ def predict(self, image: str) -> sv.Detections:
 
         detections = self.detection_model.predict(image)
 
+        if self.set_of_mark is not None:
+            label_annotator = sv.LabelAnnotator(text_position=sv.Position.CENTER)
```

Comment:
Maybe you should pass in your own Supervision annotator to give you control over how the marks will be configured (eg center of mass, bbox vs label, etc). Different models may respond better to different approaches.